modules/farm_fd2/src/entrypoints/soil_disturbance/lib.submit.unit.cy.js

```diff
@@ -1,42 +1,34 @@
 import { lib } from './lib';
-import { lib as directSeedingLib } from '../direct_seeding/lib.js';
 import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
 
-function runTest(terminationValue) {
-  describe(`Test the Soil Disturbance lib submission with termination=${terminationValue}`, () => {
-    let directSeedingBroccoli = {
-      seedingDate: '1950-01-02',
-      cropName: 'BROCCOLI',
-      locationName: 'ALF',
-      beds: ['ALF-1', 'ALF-3'],
-      bedFeet: 100,
-      rowsPerBed: '3',
-      bedWidth: 60,
-      equipment: ['Tractor'],
-      depth: 6,
-      speed: 5,
-      comment: 'A comment',
-    };
-
-    let directSeedingBean = {
-      seedingDate: '1950-01-02',
-      cropName: 'BEAN',
-      locationName: 'ALF',
-      beds: ['ALF-1', 'ALF-3'],
-      bedFeet: 100,
-      rowsPerBed: '3',
-      bedWidth: 60,
-      equipment: ['Tractor'],
-      depth: 6,
-      speed: 5,
-      comment: 'A comment',
-    };
+function runTest(activePlantAsset, terminationValue) {
+  describe(`Test the Soil Disturbance lib submission with activePlantAssets=${activePlantAsset} termination=${terminationValue}`, () => {
+    // Skip the test if the configuration is invalid
+    if (!activePlantAsset && terminationValue) {
+      // Can't terminate without an active plant asset involved
+      it('Should catch invalid configuration and mark as complete', () => {
+        cy.log(
+          'Invalid configuration caught: Termination cannot be true without an active plant asset. Marking this test as complete.'
+        );
+        expect(true).to.be.true; // Mark the test as passing
```

Comment:
Seems like this is saying that this function should never be called with `activePlantAsset=false` and `terminationValue=true`.  So I think the thing to do here is to have this test fail in this situation.  Then remove your call to it at the bottom of the file.  That way if someone in the future make the call with (`false`, `true`) their test will immediately fail.  But otherwise, this will have no effect on the tests.